### ISSUE TITLE
Main thread panicked at 'called `Option::unwrap()` on a `None` value' using macOS

### ISSUE THREAD

**Issue by NRngnl** [proshunsuke/colmsg#84]
I got the following error and I used the debugger to find the problem.
I found that the problem is related to the `.DS_Store` file generated by macOS for storing attributes.
https://en.wikipedia.org/wiki/.DS_Store

When the `.DS_Store` file is generated in member's folder, the function `id_by_filename_format` cannot capture the `id` and it will panic.

Can I make a pull request NRngnl/colmsg@ff89f70?
I am not sure if I should change the functions.

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/message/saver.rs:219:17
stack backtrace:
   0: _rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::panicking::panic
   3: colmsg::message::saver::id_by_filename_format
   4: colmsg::message::saver::Saver<C>::id_dates::{{closure}}
   5: walkdir::IntoIter::push
   6: walkdir::IntoIter::handle_entry
   7: <walkdir::IntoIter as core::iter::traits::iterator::Iterator>::next
   8: <core::iter::adapters::map::Map<I,F> as core::iter::traits::iterator::Iterator>::next
   9: <alloc::vec::Vec<T> as alloc::vec::spec_from_iter::SpecFromIter<T,I>>::from_iter
  10: colmsg::message::saver::Saver<C>::save
  11: colmsg::run_hinatazaka
  12: colmsg::main
```

**Comment by proshunsuke** [comment]
Thanks for the issue.
It was not expected that files other than the message files would be put in the download directory.
So, I think it might be better to change the logic so that only message files are targeted.

Pull request is always welcome.

**Comment by NRngnl** [comment]
I created a pull request.
I tested it on my side and it fixed the problem.

Please let me know if I break something else.
Thanks.

**Comment by proshunsuke** [comment]
Released [`v3.0.2`](https://github.com/proshunsuke/colmsg/releases/tag/v3.0.2)
Thanks for your contribution.
